back/sssscs/src/main/java/com/ib/certificate/CertificateController.java

```diff
@@ -65,12 +61,12 @@ public ResponseEntity<String> makeRequest(/*@DTO(CertificateRequestCreateDto.cla
 	public ResponseEntity<String> acceptRequest(@PathVariable Long id) {
 		CertificateRequest req = certificateRequestService.findByIdAndStatus(id, Status.PENDING);
 		
-		User issuee = userService.findById(1L); // TODO: Fetch user ID from token.
-		if (issuee == null) {
+		User issuer = userService.findById(1L); // TODO: Fetch user ID from token.
+		if (issuer == null) {
 			return new ResponseEntity<>("User does not exist", HttpStatus.NOT_FOUND);
 		}
```

Comment:
Can't you get issuer from parent? Also there is code to set issuer in Certificate constructor. Redundant code?